patterns/aggregated-wildfire.php

```diff
@@ -0,0 +1,1612 @@
+<?php
+
+/**
+ * Title: Aggregated Wildfire
+ * Slug: bcgov-wordpress-block-theme-child-theme-emergency-info/aggregated-wildfire
+ * Categories: Events
```

Comment:
Are the categories working? I thought we had to add them in a hook manually.

Reply:
just fixed it...